### DISCUSSION TITLE
How to get breakpoints to set collapsed section?

### DISCUSSION THREAD

**Discussion by aryadilas** [filamentphp/filament#10711] [Help] [answered]
I want to make section collapsed in "sm" but not collapsed if bigger than "sm" , is it possible to getBreakpoint() like that ? 
this is what i think
```
->collapsed(fn () => getBreakpoint() == "sm" ? true  :  false)
```

**Comment by leandrocfe** [comment, marked as answer]
It isn't possible I think

**Comment by ArtMin96** [comment]
Try this:
[Livewire Window Size and Breakpoints](https://github.com/TinaHammar/livewire-window-size)
